### PR TITLE
Increase Coverage for Github API

### LINKED ISSUE #283: Increase backend github/api coverage
**Is your feature request related to a problem? Please describe.**
We don't have any coverage here and want it to be 75%+

```
apps/github/api/issue.py                                                            9      9      0      0     0%   3-29
apps/github/api/label.py                                                            9      9      0      0     0%   3-28
apps/github/api/organization.py                                                     9      9      0      0     0%   3-29
apps/github/api/release.py                                                          9      9      0      0     0%   3-28
apps/github/api/repository.py                                                       9      9      0      0     0%   3-27
apps/github/api/urls.py                                                            14     14      0      0     0%   3-19
apps/github/api/user.py                                                             9      9      0      0     0%   3-29
```

### PR DESCRIPTION
**Pull request by Naveen-Pal**
Resolves #283 

Added tests for github api.
<br>
Screenshot 2024-12-31 175915
<br>
Screenshot 2024-12-31 175929
<br>
I have also modified github/api/label.py to prevent below error. Please review that change. 
"nest_created_at" is taken from TimestampedModel (app/common/models)
<br>
Screenshot 2024-12-31 175344

### CODE REVIEW

File: backend/apps/github/api/label.py [hunk 1]
```diff
@@ -15,8 +15,8 @@ class Meta:
             "name",
             "description",
             "color",
-            "created_at",
-            "updated_at",
+            "nest_created_at",
+            "nest_updated_at",
```
Comment (arkid15r) [inline backend/apps/github/api/label.py]: It seems we don't have that data. Just remove it, users don't need our internal timestamps.

File: backend/tests/github/api/issue_tests.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            "title": "Test Issue 2",
+            "body": "This is a test issue 2",
+            "state": "closed",
+            "url": "http://example.com/issues/2",
+            "created_at": "2024-12-29T00:00:00Z",
+            "updated_at": "2024-12-30T00:00:00Z",
+        },
+    ],
+)
+def test_issue_serializer(issue_data):
```
Comment (arkid15r) [inline backend/tests/github/api/issue_tests.py]: Make it a part of a class (other tests too)

**Review by arkid15r — CHANGES_REQUESTED** [review]
Looks good, let's merge it after addressing the following:

**Comment by Naveen-Pal** [comment]
@arkid15r I have added test class and removed timestamps.

File: backend/tests/github/api/organization_tests.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                "name": "Microsoft",
+                "login": "microsoft",
+                "company": "Microsoft Corporation",
+                "location": "Redmond, WA",
+                "created_at": "2024-12-29T00:00:00Z",
+                "updated_at": "2024-12-30T00:00:00Z",
+            },
+        ],
+    )
+    @patch("apps.github.models.organization.Organization.objects.filter")
```
Comment (arkid15r) [inline backend/tests/github/api/organization_tests.py:31]: Why do we need this patch here?
Could you add a comment explaining that?

Comment (Naveen-Pal) [reply backend/tests/github/api/organization_tests.py:31]: If I don't patch, the database will be queried.
image

**Review by arkid15r — APPROVED** [review]
LGTM 👍

**Pull request merged**